src/Appwrite/Transformation/Transformation.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Appwrite\Transformation;
+
+class Transformation
+{
+    public function __construct(protected Adapter $adapter)
```

Comment:
I would expect this class to accept an array of transformers in specific order and allow them to run one after the other. $transformation->add(Adapter $adapter) kind of thing.